Resources/Prototypes/_Sunrise/Catalog/ert_uplink_catalog.yml

```
@@ -496,6 +566,30 @@
+  cost:
+    Bluecrystal: 4
+  categories:
+    - NTexplosive
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 5
+
+- type: listing
+  id: NTUplinkSunriseWeaponGrenadeLauncherGL70Flash
+  productEntity: WeaponGrenadeLauncherGL70Flash
+  icon: { sprite: _Sunrise/Interface/Misc/nt_icons.rsi, state: gl79 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)